main/boofcv-io/src/main/java/boofcv/io/image/ConvertRaster.java

```diff
@@ -273,13 +274,27 @@ public static int getOffset( WritableRaster raster ) {
 			return 0;
 
 		try {
-			Method m = raster.getClass().getMethod("getDataOffset", int.class);
 			int min = Integer.MAX_VALUE;
+			Method m = raster.getClass().getMethod("getDataOffset", int.class);
+			m.setAccessible(true);
 			for (int i = 0; i < raster.getNumDataElements(); i++) {
 				min = Math.min(min, (Integer)m.invoke(raster, i));
 			}
 			return min;
-		} catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
+		} catch (NoSuchMethodException | IllegalAccessException | InaccessibleObjectException | InvocationTargetException e) {
+
+			SampleModel sm = raster.getSampleModel();
+			if (sm instanceof ComponentSampleModel) {
+				ComponentSampleModel csm = (ComponentSampleModel)sm;
+				// according to https://github.com/openjdk/jdk/blob/master/src/java.desktop/share/classes/sun/awt/image/ByteInterleavedRaster.java
```

Comment:
Interestingly, as of writing this comment, the linked implementation has `getDataOffset` declared as a public method.